src/main/java/org/sagebionetworks/web/client/widget/accessrequirements/createaccessrequirement/CreateManagedACTAccessRequirementStep3.java

```diff
@@ -249,9 +58,15 @@ public Widget asWidget() {
   @Override
   public void setModalPresenter(ModalPresenter modalPresenter) {
     this.modalPresenter = modalPresenter;
-    modalPresenter.setTitle(
-      "People with permission in this Access Requirement"
-    );
-    modalPresenter.setPrimaryButtonText(DisplayConstants.FINISH);
+    modalPresenter.setTitle("Access Requirement Permissions");
+    modalPresenter.setPrimaryButtonText(DisplayConstants.SAVE_BUTTON_LABEL);
```

Comment:
Updated based on [the design](https://www.figma.com/file/vLusb6uSfhx45OyFx5IHwy/(XDM)-Extensible-Data-Management-Comps?type=design&node-id=7674-6758&mode=dev)